catalog-info.yaml

```diff
@@ -0,0 +1,29 @@
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-terraform-provider-elasticstack-release
+  description: Buildkite Pipeline for releasing Terraform provider Elasticstack
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/terraform-provider-elasticstack-release
+
+spec:
+  type: buildkite-pipeline
+  owner: group:control-plane-stateful-applications
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: terraform-provider-elasticstack / release
+      description: Terraform provider Elasticstack - release
+    spec:
+      repository: elastic/terraform-provider-elasticstack
+      pipeline_file: ".buildkite/release.yml"
+      teams:
+        control-plane-stateful-applications:
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
```

Comment:
So I wouldn't be able to trigger a release with this permission set (which in theory should be fine). I wonder if we might want to grant manage build to serverless-core-arch or something as well. 

It's not clear what the plans for triggering this are, if we're expecting to have this pipeline triggered when a vX.Y branch is pushed then I guess it shouldn't matter too much.

Reply:
@tobio, I fixed perms for your group.